review/migrations/0001_initial.py

```diff
@@ -1,6 +1,7 @@
-# Generated by Django 4.2 on 2023-04-17 17:12
+# Generated by Django 4.2 on 2023-04-23 08:56
```

Comment:
I think you should run makemaigrations with model changes and add the extra init file, am i wrong?

Reply:
Yes, this is an updated version from yesterday with updated FK fields that come from `account` app